src/router/router.ts

```diff
@@ -152,7 +151,7 @@ export class Router extends EventEmitter<Handlers> {
     this._transaction.push(RouteAction.dropSearchParams());
   }
 
-  public commit(_commitOpts?: CommitOptions): this {
+  public commit(): this {
```

Comment:
I don't see much advantages on such changes. Feels like there should be a separate rule (or options for existing rules) which could change this point: its wrong to remove unused function args because obviously they can be a part of function signatures.